svelte-web-frontend/src/lib/model/settings.ts

```diff
@@ -1,14 +1,26 @@
 import { dev } from '$app/environment';
 import { writable } from 'svelte/store';
 
-export const debugMode = writable(dev);
+export const debugMode = writable(localStorage.debugMode === 'true');
 
-export const pauseInPracticePage = writable(false);
+export const pauseInPracticePage = writable(localStorage.pauseInPracticePage === 'true');
 
-export const debugPauseDurationSecs = writable(1.0);
+export const debugPauseDurationSecs = writable( parseFloat(localStorage.debugPauseDurationSecs) || 1.0);
 
-export const evaluation_GoodBadTrialThreshold = writable(4.0);
+export const evaluation_GoodBadTrialThreshold = writable(parseFloat(localStorage.evaluation_GoodBadTrialThreshold) || 4.0);
 
-export const feedback_YellowThreshold = writable(3.0);
+export const feedback_YellowThreshold = writable(parseFloat(localStorage.feedback_YellowThreshold) || 3.0);
 
-export const feedback_GreenThreshold = writable(4.2);
\ No newline at end of file
+export const feedback_GreenThreshold = writable(parseFloat(localStorage.feedback_GreenThreshold) || 4.2);
```

Comment:
Can we refactor so we only define the hardcoded default values in one place (the DRY principle). This will make it easier to update / maintain the default values going forward, so we won't have to remember to update it in two places.

I suggest defining a const frozen object at the beginning of this file that you can refer to when creating the writable stores and in the reset function. Like this:
```
const DEFAULT_SETTINGS = Object.freeze({
  debugMode:  true,
  pauseInPracticePage: false,
  ...
});
```